### DISCUSSION TITLE
"ATLAS sets stringent limits on the existence of supersymmetric dark matter particles"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#171] [Gravitation Models]
"ATLAS sets stringent limits on the existence of supersymmetric dark matter particles"
https://phys.org/news/2023-10-atlas-stringent-limits-supersymmetric-dark.html

More parameter space excluded, no dark matter found... yet again.

**Comment by bgreenman** [comment]
Your statement is correct

**Reply by Francois-Zinserling** [reply]
SUSY has died several deaths already. To try and use supersymmetry to find another (dark matter is dead) particle is not going to miraculously revive both theories. 
The desperation shows.

**Reply by bgreenman** [reply]
your right
